changelog/unreleased/kong/fix-request-transformer-uri-replace.yml

```
@@ -0,0 +1,4 @@
+message: |
+  **reqeust-transformer**: Fixed an issue where the URI captures are unavailable when an optional group is absent.
```

```suggestion
  Fixed an issue where the URI captures are unavailable when an optional group is absent.
```
We did not change the code of plugin, so need not to add plugin info in changelog.